src/commands/index.ts

```diff
@@ -102,27 +102,27 @@ function resetProjectDetectionDefaultsCommand(): void {
 }
 
 function toggleAngularPresetCommand(): void {
-  togglePreset('angular', 'ngPreset', false, false);
+  togglePreset(models.PresetNames[models.PresetNames.angular], 'ngPreset', false, false);
```

Comment:
In order to make the first parameter less verbose why not to pass `models.PresetNames.xxx` and resolve it to string inside `togglePreset`?

Reply:
OK.